### DISCUSSION TITLE
Backup/Restore Concept

### DISCUSSION THREAD

**Discussion by RummsMurmel** [Shuffle/Shuffle#1054] [Q&A]
Hello,

now that we want to use Shuffle productively, a few questions arise about backup and restore.

Which components should be backed up to restore all data such as workflows and authentication in case of a DR on a completely new VM?

A listing in the documentation would be great.

So far I am guessing the following based on the general documentation:

- Encryption Key - ${SHUFFLE_ENCRYPTION_MODIFIER}
- File Location - ${SHUFFLE_FILE_LOCATION}
- DB-Volume - ${DB_LOCATION}

Is it enough to secure the volume of the opensearch container? Or is it better to create snapshots of the Opensearch index using Opensearch API calls (https://opensearch.org/docs/1.2/opensearch/snapshot-restore/)?

And: what would be the correct procedure for a restore?

Thanks a lot!

**Comment by frikky** [comment]
This totally makes sense, and I added a documentation issue for it here: https://github.com/Shuffle/Shuffle-docs/issues/166

Of the bat, I think you're right with the three general locations being where you should start. The File location is not necessary, but great to have if you want to keep old files.
